pages/mobile/base.py

```diff
@@ -11,36 +11,47 @@
 
 class Base(Page):
 
+    _body_locator = (By.TAG_NAME, 'body')
+    _menu_items_locator = (By.CSS_SELECTOR, 'nav a')
+    _menu_button_locator = (By.ID, 'menu-button')
+
     @property
-    def header(self):
-        return Base.HeaderRegion(self.testsetup)
+    def is_menu_exposed(self):
+        return 'exposed' in self.selenium.find_element(*self._body_locator).get_attribute('class')
 
-    class HeaderRegion(Page):
+    @property
+    def menu_items(self):
+        return [self.MenuItem(self.testsetup, web_element) for web_element in self.selenium.find_elements(*self._menu_items_locator)]
 
-        _dropdown_expanded_menu_locator = (By.CSS_SELECTOR, 'div.moz-menu')
-        _menu_items_locator = (By.CSS_SELECTOR, '.menu-items li')
-        _menu_button_locator = (By.CSS_SELECTOR, '.tab > a')
+    def click_menu(self):
+        self.selenium.find_element(*self._menu_button_locator).click()
 
-        def click_header_menu(self):
-            self.selenium.find_element(*self._menu_button_locator).click()
+    def click_menu_item(self, item_name):
+        if not self.is_menu_exposed:
```

Comment:
Yeah, I've removed that now. I thought I needed it as I was going through the tests, but it turned out that it wasn't used.